src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Linn.Purchasing.Facade.ResourceBuilders
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Linn.Common.Authorisation;
+    using Linn.Common.Facade;
+    using Linn.Common.Reporting.Models;
+    using Linn.Common.Resources;
+    using Linn.Purchasing.Domain.LinnApps;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+    using Linn.Purchasing.Resources;
+
+    public class PurchaseOrderResourceBuilder : IBuilder<PurchaseOrder>
+    {
+        private readonly IAuthorisationService authService;
+
+        public PurchaseOrderResourceBuilder(IAuthorisationService authService)
+        {
+            this.authService = authService;
+        }
+
+        public PurchaseOrderResource Build(PurchaseOrder entity, IEnumerable<string> claims)
+        {
+            var detail = entity.Details.First(x => x.Line == 1);
+            {
+                return new PurchaseOrderResource
+                           {
+                                OrderNumber = entity.OrderNumber,
+                                Cancelled = entity.Cancelled,
+                                DocumentType = entity.DocumentType,
+                                DateOfOrder = entity.OrderDate,
+                                Overbook = entity.Overbook,
+                                OverbookQty = entity.OverbookQty,
+                                SupplierId = entity.SupplierId,
+                                Links = this.BuildLinks(null, claims).ToArray()
+                           };
+            }
+        }
+
+        public string GetLocation(PurchaseOrder p)
+        {
+            return $"/purchasing/overbook/{p.OrderNumber}";
```

Comment:
i think the location should be /purchasing/purchase-orders/{orderNumber} or similar. 

The overbook should be a separate rel with an href something like 
/purchasing/purchase-orders/{orderNumber/over-book

Reply:
As per first review 

the location should be /purchasing/purchase-orders/{orderNumber} or similar.

The over book should be a separate rel with an href something like
/purchasing/purchase-orders/{orderNumber/over-book